src/services/api/src/queue/handle.py

```diff
@@ -27,28 +25,11 @@ def __init__(self, model_key: str):
 
         self.model_key = model_key
 
-        app_name = self.app_name
-
-        # We reimplement `serve.get_app_handle` to manually look up the ingress name.
-        # That way we can add a timeout and avoid a deadlock.
-        client = _get_global_client()
-        deployment_name = ray.get(
-            client._controller.get_ingress_deployment_name.remote(app_name),
-            timeout=int(os.environ.get("COORDINATOR_HANDLE_TIMEOUT_S", "5")),
-        )
-
-        if deployment_name is None:
-            raise RayServeException(f"Application '{app_name}' does not exist.")
-
-        self.handle = DeploymentHandle(deployment_name, app_name)
-        # We need the namespace of the deployment as otherwise we can't lookup its model actor.
-        # We use completion of this call to determine the deployment is finished initializing.
-        self.namespace = self.handle.ready.remote()
-
-    @property
-    def app_name(self):
-        """Canonical Ray Serve app name derived from `model_key`."""
-        return f"Model:{slugify(self.model_key)}"
+        try:
+            actor = ray.get_actor(self.actor_name, namespace="NDIF")
+            self.ready_future = actor.__ray_ready__.remote()
+        except Exception:
```

Comment:
Maybe log exception as error? There might be useful diagnosable information to extract